src/bin/__test__/cli.test.ts

```diff
@@ -0,0 +1,37 @@
+import run from '../cli'
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+test('cli prints help', () => {
+  const logSpy = jest.spyOn(global.console, 'log').mockImplementation(jest.fn())
+  let exitCode
+  //@ts-expect-error never type can't work here
+  const exitSpy = jest.spyOn(process, 'exit').mockImplementation((code?: number): never => {
```

Comment:
How about `void` instead of `never`?

Reply:
the original function is typed with `never` so it gave me a type mismatch error if I said it the return type was `void`